src/NovelRT/Atom.cpp

```diff
@@ -2,89 +2,90 @@
 // for more information.
 
 #include <NovelRT/Atom.h>
+#ifndef __TBB_PREVIEW_MUTEXES
+#define __TBB_PREVIEW_MUTEXES 1
+#endif
+#include <NovelRT/Exceptions/Exceptions.h>
+#include <mutex>
```

Comment:
Do we need `std::mutex` if we're now using `tbb::mutex`?

Reply:
for `std::scoped_lock`

Reply:
Coolio, ty for confirming